src/chrome/content/scaffold/scaffold.js

```diff
@@ -955,11 +955,10 @@ var Scaffold = new function() {
 	function updateSelectedTests() {
 		_clearOutput();
 		var listbox = document.getElementById("testing-listbox");
-		var items = listbox.selectedItems.slice();
+		var items = [...listbox.selectedItems];
 		if(!items || items.length == 0) return false; // No action if nothing selected
-		var i;
 		var tests = [];
-		for (i in items) {
+		for (var i=0; i<items.length; i++) {
 			items[i].getElementsByTagName("listcell")[1].setAttribute("label", "Updating");
```

Comment:
I cannot exactly follow. AFAIK it is important that `items` are a separate copy of the array `listbox.selectedtems` and by changing one of them, the other copy will not change at the same time. The deprecated `.slice()` and the new spread solution should be equivalent. I don't see how the same is possible without an intermediate array.